home/models.py

```diff
@@ -41,24 +68,54 @@ def render_basic(self, value, context=None):
         pass
 
 
-def get_choices():
-    if SiteSettings.objects.first():
-        return [
-            (choice, choice)
-            for choice in SiteSettings.objects.first().content_variations_options
-        ]
+def get_valid_profile_values(field):
+    sitesettings = Site.objects.filter(is_default_site=True).first().sitesettings
```

Comment:
get_object_or_404 might be better here? because if this does not exist, then this will throw an error NoneType has no sitesettings. 

or maybe this would not break if you do like

SiteSettings.for_site(Site.object.get(default_site=True)

only one site can be default site. so the get will not fail unless there is no site

Reply:
coolio. I don't think it's even possible to get here if there isn't a default site set but rather safe than sorry